diff --git a/node_modules/@actions/glob/lib/internal-pattern.js b/node_modules/@actions/glob/lib/internal-pattern.js
index 0b68b54..f5354dd 100644
--- a/node_modules/@actions/glob/lib/internal-pattern.js
+++ b/node_modules/@actions/glob/lib/internal-pattern.js
@@ -2,10 +2,9 @@ import * as os from 'os';
 import * as path from 'path';
 import * as pathHelper from './internal-path-helper.js';
 import assert from 'assert';
-import minimatch from 'minimatch';
+import { Minimatch } from 'minimatch';
 import { MatchKind } from './internal-match-kind.js';
 import { Path } from './internal-path.js';
-const { Minimatch } = minimatch;
 const IS_WINDOWS = process.platform === 'win32';
 export class Pattern {
     constructor(patternOrNegate, isImplicitPattern = false, segments, homedir) {
